src/uct/base/uct_md.c

```diff
@@ -263,6 +263,10 @@ ucs_status_t uct_md_config_read(uct_component_h component,
     uct_config_bundle_t *bundle = NULL;
     ucs_status_t status;
 
+    if (component->md_config.table == NULL) {
+        goto out;
+    }
```

Comment:
which test/ucp code is calling md_open ith NULL config table?
also, better :
*config_p = NULL;
goto out;

and move out: label to return UCS_OK

Reply:
> which test/ucp code is calling md_open ith NULL config table?
> also, better :
> *config_p = NULL;
> goto out;
> 
> and move out: label to return UCS_OK

https://github.com/openucx/ucx/blob/c1560bd8488b6caeada4af9a05f3ad4531b916de/test/gtest/uct/uct_test.cc#L156

Reply:
need to change uct test, to check UCT_COMPONENT_ATTR_FIELD_MD_RESOURCE_COUNT and read md config only if it's >0

Reply:
seems working, do you prefer to remove changes in MD?

Reply:
yes